doc/tools/makerst.py

```diff
@@ -369,6 +371,8 @@ def make_rst_class(class_def, state, dry_run, output_dir):  # type: (ClassDef, S
     if dry_run:
         f = open(os.devnull, "w", encoding="utf-8")
     else:
+        # Create the output folder recursively if it doesn't already exist.
+        os.makedirs(output_dir, exist_ok=True)
```

Comment:
Can't this be done only once instead of redoing it for each class? All classes are in the same output folder.